sim/core/spell_result.go

```diff
@@ -76,10 +76,6 @@ func (spell *Spell) RangedAttackPower(target *Unit) float64 {
 		target.PseudoStats.BonusRangedAttackPowerTaken
 }
 
-func (spell *Spell) BonusWeaponDamage() float64 {
```

Comment:
So it is gone :) Ignore the related remark about it being not longer necessary elsewhere.